solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md

```diff
@@ -91,11 +91,9 @@ class Solution {
 class Solution {
 public:
     int removeDuplicates(vector<int>& nums) {
-        int i = 0;
-        for (int& num : nums)
-            if (i < 1 || num != nums[i - 1])
-                nums[i++] = num;
-        return i;
+        auto ret = std::ranges::unique(nums);
+        nums.erase(ret.begin(), ret.end());
```

Comment:
Note that this is equivalent to, before C++20,
```cpp
nums.erase(unique(nums.begin(), nums.end()), nums.end());
```

Reply:
We can add a new C++ implenentation.

### **Java**

...

### **C++**

```cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        for (int& num : nums)
            if (i < 1 || num != nums[i - 1])
                nums[i++] = num;
        return i;
    }
};
```

```cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        nums.erase(unique(nums.begin(), nums.end()), nums.end());
        return nums.size();
    }
};
```

### **Go**

...

Reply:
And this implementation should be added to both README.md and README_EN.md files.

Reply:
@Bernhard-L

Reply:
fine by me

Reply:
@Bernhard-L This was just a comment. I'd also love C++20 features here too! Sorry for this